shell/pages/c/_cluster/settings/links.vue

```diff
@@ -0,0 +1,203 @@
+  components: {
+    KeyValue,
+    Loading,
+    AsyncButton,
+    Banner,
+  },
+  async fetch() {
+    try {
+      const { uiIssuesSetting, uiCommunitySetting } = await allHash({
+        uiIssuesSetting:    this.$store.dispatch('management/find', { type: MANAGEMENT.SETTING, id: SETTING.ISSUES }),
+        uiCommunitySetting: this.$store.dispatch('management/find', { type: MANAGEMENT.SETTING, id: SETTING.COMMUNITY_LINKS })
+      });
+
+      this.uiIssuesSetting = uiIssuesSetting;
+      this.uiCommunitySetting = uiCommunitySetting;
+    } catch {}
+
+    try {
+      const defaultIssueLink = { key: 'File an issue', value: this.uiIssuesSetting.value } || DEFAULT_SUPPORT_LINK;
+      const defaultLinks = [...DEFAULT_CUSTOM_LINKS, defaultIssueLink, ...COMMUNITY_LINKS];
+
+      this.uiCustomLinks = await fetchOrCreateSetting(this.$store, SETTING.UI_CUSTOM_LINKS, JSON.stringify(defaultLinks));
+
+      await this.deprecateIssueLinks();
+    } catch {}
+
+    const sValue = this.uiCustomLinks?.value || JSON.stringify(DEFAULT_CUSTOM_LINKS);
+
+    this.value = JSON.parse(sValue);
+  },
+  data() {
+    return {
+      uiCustomLinks: {},
+      bannerVal:     {},
+      value:         {},
+      errors:        [],
+    };
+  },
+  computed: {
+    mode() {
+      const schema = this.$store.getters[`management/schemaFor`](MANAGEMENT.SETTING);
+
+      return schema?.resourceMethods?.includes('PUT') ? _EDIT : _VIEW;
+    },
+
+    defaultLinks: {
+      get(issueLink) {
+        if ( issueLink ) {
+          return [...DEFAULT_CUSTOM_LINKS, issueLink, ...DEFAULT_CUSTOM_LINKS];
+        }
+
+        return [...DEFAULT_CUSTOM_LINKS, DEFAULT_SUPPORT_LINK, ...DEFAULT_CUSTOM_LINKS];
+      }
+    }
+
+  },
+  methods: {
+    useDefaults() {
+      const nonCommercialRancherLinks = this.isCommercial ? [] : COMMUNITY_LINKS;
+
+      this.value = [...DEFAULT_CUSTOM_LINKS, DEFAULT_SUPPORT_LINK, ...nonCommercialRancherLinks];
+    },
+
+    cancel() {
+      this.done();
+    },
+
+    deprecateIssueLinks() {
+      if (this.uiIssuesSetting.value || this.uiIssuesSetting.value) {
+        this.uiIssuesSetting.value = '';
+        this.uiCommunitySetting.value = '';
+
+        return Promise.all([
+          this.uiIssuesSetting.save(),
+          this.uiIssuesSetting.save()
+        ]);
+      }
+    },
+
+    async save(btnCB) {
+      this.uiCustomLinks.value = JSON.stringify(this.value);
+      this.errors = [];
+      try {
+        await this.uiCustomLinks.save();
+        btnCB(true);
+      } catch (err) {
+        this.errors.push(err);
+        btnCB(false);
+      }
+    },
+  }
+};
+</script>
+<template>
+  <Loading v-if="$fetchState.pending" />
+  <div v-else>
+    <h1 class="mb-20">
+      {{ t("customLinks.label") }}
+    </h1>
+    <div>
+      <label class="text-label">
+        {{ t(`customLinks.description`, {}, true) }}
+      </label>
+      <!-- Incremental Loading -->
+      <div class="ui-links-setting">
+        <KeyValue
```

Comment:
Should the user be able to add more entries to this collection via the  `Add Label` button?